QRCoder/QRCodeGenerator.cs

```diff
@@ -745,14 +745,22 @@ bool IsUtf8()
             }
         }
 
+        private static readonly Encoding _iso88591ExceptionFallback = Encoding.GetEncoding(28591, new EncoderExceptionFallback(), new DecoderExceptionFallback()); // ISO-8859-1
         /// <summary>
         /// Checks if the given string can be accurately represented and retrieved in ISO-8859-1 encoding.
         /// </summary>
         private static bool IsValidISO(string input)
         {
-            var bytes = Encoding.GetEncoding("ISO-8859-1").GetBytes(input);
-            var result = Encoding.GetEncoding("ISO-8859-1").GetString(bytes);
-            return String.Equals(input, result);
```

Comment:
Here we eliminated conversion from UTF-16 to ISO-8859-1 and then back again (two conversions).